File: app/src/main/java/com/example/billeteramodulo4/Home_Page.java
package com.example.billeteramodulo4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;


public class Home_Page extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.home_page);


        // Abrir nueva activity

        ImageButton boton = findViewById(R.id.sendhome);

        boton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                //ir a activity 4
                Intent i = new Intent(Home_Page.this, Home_Page_Empty_Case.class);
                startActivity(i);
            }
        });


    }
}

File: app/src/main/java/com/example/billeteramodulo4/HomePage.kt
package com.example.billeteramodulo4

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import androidx.recyclerview.widget.LinearLayoutManager
import com.example.billeteramodulo4.adapter.ContactoAdapter
import com.example.billeteramodulo4.databinding.FragmentHomePageBinding

class HomePage : Fragment() {

    private lateinit var binding: FragmentHomePageBinding


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {

        binding = FragmentHomePageBinding.inflate(inflater, container, false)
        val root = binding.root

        initRecyclerView()



        binding.btnenviadinerohomepage.setOnClickListener {
            findNavController().navigate(R.id.action_homePage_to_sendMoney)
        }

        binding.ingresadinerohomepage.setOnClickListener {
            findNavController().navigate(R.id.action_homePage_to_requestMoney)
        }

        binding.profileuser.setOnClickListener {
            findNavController().navigate(R.id.action_homePage_to_profilePage)
        }
        return root

    }

    private fun initRecyclerView() {
        val recyclerView = binding.recyclerView
        recyclerView.layoutManager = LinearLayoutManager(context)
        recyclerView.adapter = ContactoAdapter(DatosContactoProveedor.contactoList)
    }

}

